pkgs/by-name/os/osc-cli/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "osc-cli";
+  version = "1.11.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "outscale";
+    repo = "osc-cli";
+    rev = "v${version}";
+    hash = "sha256-7WXy+1NHwFvYmyi5xGfWpq/mbVGJ3WkgP5WQd5pvcC0=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    bandit
+    defusedxml
+    fire
+    mypy
+    mypy-extensions
+    pylint
+    pytest
+    requests
+    types-requests
+    typing-extensions
+    xmltodict
+  ];
+
+  nativeCheckInputs = [
+    python3.pkgs.pytest
```

Comment:
Please use pytestCheckHook

Reply:
Turns out tests require a working access and secret keys to work so I disabled them completely.